packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -392,7 +393,7 @@ export default class SalesforceCRMService implements CRM {
 
     if (!records.length) records = results.records as ContactRecord[];
 
-    if (includeOwner || forRoundRobinSkip) {
+    if ((includeOwner || forRoundRobinSkip) && !this.attendeeHasFreeEmailDomain(emails[0])) {
```

Comment:
nit: I think instead of doing emails[0]. We should have it stored as booker or similar and then use that variable to make code more readable and consistent

Reply:
Emails come from the `CalendarEvent.attendees` type which is used throughout the code base. If we were to make this change I'd want to do it in a separate PR.

Reply:
Oh I meant, storing it in a variable, so at the top of the fn, we should do
`const booker = attendees[0]` and then we use `booker` everywhere which is much more logical then using attendees[0]